livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -195,6 +195,13 @@ class _ContentPtr(TypedDict):
 DEFAULT_INPUT_AUDIO_TRANSCRIPTION = InputTranscriptionOptions(model="whisper-1")
 
 
+class _NOTSET:
```

Comment:
Let's use the NOT_GIVEN we already have here:
https://github.com/livekit/agents/blob/4a9b845117e0569ff8b221dc2c83e72e728ab853/livekit-agents/livekit/agents/types.py#L30